### PR TITLE
Securityimplication

### PR DESCRIPTION
**Pull request by DianeLiu2019**
documentation for security implications

### CODE REVIEW

File: _pages/en/technical-platform.md [hunk 1]
```diff
@@ -86,7 +86,15 @@ More detailed Azure DevOps Pipeline doc: [Azure Pipeline](../assets/docs/azure-p
 
 ## Security Implications
 
-- Things we have done to make it more secure.
+- Using GraphQL mobible friendly API with strong input validation to prevent hacking the database.
```
Comment (danprime) [inline _pages/en/technical-platform.md]: Spelling mistake here?
Is mobile-friendly really the qualifier we want to put here?

File: _pages/en/technical-platform.md [hunk 2]
```diff
@@ -86,7 +86,15 @@ More detailed Azure DevOps Pipeline doc: [Azure Pipeline](../assets/docs/azure-p
...
+- Using containers to minimize attack surface--less code bug, less chance to be attacked.
```
Comment (danprime) [inline _pages/en/technical-platform.md]: the second half (less code bugs, less chance to be attacked) explains the benefits not so much how containers minimize attack surface (e.g. minimal OS? ports closed by default? etc.)

File: _pages/en/technical-platform.md [hunk 3]
```diff
@@ -86,7 +86,15 @@ More detailed Azure DevOps Pipeline doc: [Azure Pipeline](../assets/docs/azure-p
 
 ## Security Implications
 
-- Things we have done to make it more secure.
+- Using GraphQL mobible friendly API with strong input validation to prevent hacking the database.
+- Using containers to minimize attack surface--less code bug, less chance to be attacked. 
+- Continuous deployement let us fix security problems quickly- pipleline has "npm install",  will check all the dependence security issue. 
+- Scrambling personal information in both protypes before saving to database.
+- Only the delivery team members can access to the data which gathered from protypes testing.
```
Comment (danprime) [inline _pages/en/technical-platform.md]: Suggest reviewing this to be more clear. Right now, CDS delivery team members can't access data only RCMP team members. Need a way to phrase it that only authorized people can access data?

File: _pages/en/technical-platform.md [hunk 4]
```diff
@@ -86,7 +86,15 @@ More detailed Azure DevOps Pipeline doc: [Azure Pipeline](../assets/docs/azure-p
 
 ## Security Implications
 
-- Things we have done to make it more secure.
+- Using GraphQL mobible friendly API with strong input validation to prevent hacking the database.
+- Using containers to minimize attack surface--less code bug, less chance to be attacked. 
+- Continuous deployement let us fix security problems quickly- pipleline has "npm install",  will check all the dependence security issue. 
+- Scrambling personal information in both protypes before saving to database.
+- Only the delivery team members can access to the data which gathered from protypes testing. 
+- Only the people with permission can access to RCMP Azure.
```
Comment (danprime) [inline _pages/en/technical-platform.md]: May need to re-phrase this and change RCMP azure to Azure? Or say that access to infrastructure is based on infrastructure's rules (running the code doesn't grant access to the infrastructure?) @sleepycat @keithiancolbourne ?

File: _pages/en/technical-platform.md (hunk 1 above)
Comment (DianeLiu2019) [reply _pages/en/technical-platform.md]: fixed

File: _pages/en/technical-platform.md (hunk 2 above)
Comment (DianeLiu2019) [reply _pages/en/technical-platform.md]: i deleted the second half

File: _pages/en/technical-platform.md (hunk 3 above)
Comment (DianeLiu2019) [reply _pages/en/technical-platform.md]: good suggestion

File: _pages/en/technical-platform.md (hunk 4 above)
Comment (DianeLiu2019) [reply _pages/en/technical-platform.md]: I changed to "Only the people with permission can access to the project Azure."

**Pull request closed without merging**